bundle/manifests/saas-operator-controller-manager-metrics-monitor_monitoring.coreos.com_v1_servicemonitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
Is this `ServiceMonitor` something that comes with operator-sdk or placed manually here? At least on previous `operator-sdk` version, operator `Service` for metrics, and its `ServiceMonitor` where automatically managed by the operator itself

Reply:
As we discussed over slack, this is now generated with kustomize.